lib/awful/widget/prompt.lua

```diff
@@ -28,6 +29,9 @@ local function run(promptbox)
         exe_callback        = function (...)
             promptbox:spawn_and_handle_error(...)
         end,
+        fg_cursor           = beautiful.prompt_fg_cursor,
+        bg_cursor           = beautiful.prompt_bg_cursor,
+        font                = beautiful.prompt_font,
```

Comment:
I think those 3 theme variables should go into `awful.prompt`, not `awful.widget.prompt`. Otherwise it will be confusing.